lib/package.json

```diff
@@ -1,6 +1,12 @@
 {
+  "scripts": {
+    "lint": "eslint React.js"
```

Comment:
Should we include `ReactDOM` here too? (actually, the preferred script would check _any_ `.js` file that is in `lib` and it's not in `node_modules`).

Reply:
good point! I replaced it with `eslint *.js`, now linting all .js files in lib directory:
https://github.com/jchavarri/rroo/pull/20/files#diff-b6397f6861def56ce657a2dde39e2cbbR3